libsolidity/ast/ASTAnnotations.h

```diff
@@ -178,6 +178,7 @@ struct CallableDeclarationAnnotation: DeclarationAnnotation
 
 struct FunctionDefinitionAnnotation: CallableDeclarationAnnotation, StructurallyDocumentedAnnotation
 {
+	std::optional<uint64_t> internalFunctionID;
```

Comment:
This may be a bit nitpicky, but it `internalFunctionId` would be proper camel casing.

```suggestion
	std::optional<uint64_t> internalFunctionId;
```

Reply:
Right, I can change it. I just followed what I saw in other parts of the code, for instance https://github.com/ethereum/solidity/blob/0aa85153e56ea6effbf37d6ddde0e0946d6937ab/libevmasm/Assembly.h#L186

Reply:
Oh, we do that? :D Never mind than, leave it as is :)

Reply:
Well, it's not like Kamil reviewed *all* code in the repo just yet :-D.

Reply:
In that case we should change it, right?

Reply:
Just keep it as far as I'm concerned.